e2e/docker-compose-test.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   jellyfish:
-    image: "ghcr.io/jellyfish-dev/jellyfish:${JELLYFISH_VERSION:-0.2.0}"
+    image: "ghcr.io/jellyfish-dev/jellyfish:${JELLYFISH_VERSION:-0.4.0}"
```

Comment:
I think it can be bumped to 0.4.2 or 0.5.0-rco

Reply:
I bumped it to 0.4.2. Let's use 0.5.0 when it is official